.ci/Dockerfile

```diff
@@ -1,6 +1,20 @@
 
 FROM cirrusci/flutter:latest
 
+RUN sudo apt-get update
+RUN sudo apt-get install -y git wget curl unzip python lsb-release sudo apt-transport-https
+
+# Add repo for gcloud sdk and install it
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | \
+    sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
+    sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+
+RUN sudo apt-get update && sudo apt-get install -y google-cloud-sdk && \
+    gcloud config set core/disable_usage_reporting true && \
+    gcloud config set component_manager/disable_update_check true
+
```

Comment:
In general, fewer RUNs could reduce Docker image size.

Reply:
I combined a couple of the run commands.

If we really wanted to be efficient we'd probably be building the Docker and putting it in the cloud like the flutter/flutter repo does: https://github.com/flutter/flutter/blob/master/.cirrus.yml#L2